pkgs/applications/office/fantasia-archive/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, fetchzip
+, appimageTools
+, makeDesktopItem
+}:
+
+let
+  pname = "fantasia-archive";
+  version = "0.1.10";
+
+  archive = fetchzip {
+    url = "https://github.com/Elvanos/fantasia-archive/releases/download/v${version}/Fantasia.Archive-${version}-linux-appImage.zip";
+    hash = "sha256-Ls+7wmj2qFoppiHJwt9Uan+9irFvzHxEWjIWrA9Nw2w=";
+  };
+  src = "${archive}/Fantasia.Archive-${version}-linux-appImage.AppImage";
+  appimageContents = appimageTools.extractType2 {
+    inherit pname src version;
+  };
+
+  description = "100% free, powerful & feature-rich offline worldbuilding tool that runs on your computer!";
```

```suggestion
  description = "100% free, powerful & feature-rich offline worldbuilding tool that runs on your computer";
```
Package short descriptions shouldn't have ending punctuation.

---

Also... this description feels like it's written by a marketing department. Maybe we should consider rewriting in the future.

---

Yeah definitely, though I couldn't find any other official sufficiently long descriptions. To be fair, it being run locally is a pretty big distinguishing factor, as the only similar software I know of are all online services (Campfire Blaze, World Anvil).